main.go

```diff
@@ -252,13 +254,13 @@ func printVersionInfo() {
 }
 
 // TODO: once we get rid of any packages that need an unmarshaller, remove this.
-type UnmarshallerOut struct {
+type unmarshallerOut struct {
```

Comment:
can we use https://github.com/xmidt-org/arrange to provide this functionality?

Reply:
Jasmine and I discussed it. It's not quite compatible with the `themis/config` package so there's no easy way today.